### PR TITLE
Minor adjustments on calendar print stylesheets

### PR DESCRIPTION
**Pull request by bp323**
**Week view:**
* [x] Can we add current week to the title? "My timetable Feb. 26 - Mar. 03"
* [x] Can we expand calendar to full screen / page height?

** Month view**
Good as it is

**Day view**
* [x] Can we expand to 100% height?
* [x] Can we include the date of the day show in the title 23 February, 20145?

my_timetable_pdf__1_page_

my_timetable_pdf__1_page_

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/gh/js/views/gh.calendar.js [hunk 1]
```diff
@@ -287,6 +294,18 @@ define(['gh.core', 'moment', 'clickover'], function(gh, moment) {
         $('#gh-toolbar-label-term').html(label);
     };
 
+    /**
+     * Set the document title to the currently selected date range
+     */
```
Comment (Coenego) [inline shared/gh/js/views/gh.calendar.js]: `@private` needed

**Comment by Coenego** [comment]
Minor issue left. Reassigning to @bp323

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

**Pull request merged**